Frontend-Angular/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "todolist.app",
+  "version": "0.1.0",
```

Comment:
remove the `package-lock.json` and let''s see if candidate will include it in source?